src/SFA.DAS.EmployerFinance/Queries/GetAccountFinanceOverview/GetAccountFinanceOverviewQueryHandler.cs

```diff
@@ -43,13 +43,15 @@ public async Task<GetAccountFinanceOverviewResponse> Handle(GetAccountFinanceOve
            
             var currentBalance = await GetAccountBalance(query.AccountId.Value);
             var earliestFundsToExpire = await GetExpiringFunds(query.AccountId.Value);
+            var paymentsForLastYear = await GetAllPaymentsForLastYear(query.AccountId.Value);
```

Comment:
variable and method name suggest this is a list of 'Payment' objects

Reply:
Renamed to better reflect purpose